src/v2/Apps/Artists/Components/ArtistsLetterNav.tsx

```diff
@@ -0,0 +1,52 @@
+import { Box, BoxProps, color, space, Text } from "@artsy/palette"
+import React from "react"
+import styled from "styled-components"
+import { RouterLink } from "v2/Artsy/Router/RouterLink"
+
+const LETTERS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ".split("")
+
+const Container = styled(Box)`
+  display: flex;
+  flex-wrap: wrap;
+  align-items: flex-start;
+  justify-content: flex-start;
+`
+
+const Letter = styled(RouterLink)`
+  display: block;
+  padding: ${space(0.5)}px ${space(1)}px;
+  text-align: center;
+  text-decoration: none;
+  border-radius: 2px;
+
+  &.active {
+    color: ${color("black100")};
+    background-color: ${color("black5")};
+    font-weight: bold;
+  }
+`
+
+interface ArtistsLetterNavProps extends BoxProps {}
+
+export const ArtistsLetterNav: React.FC<ArtistsLetterNavProps> = ({
+  ...rest
```

Comment:
(nonblocking) why not define an explicit interface? `...rest` can be a bit hard to decipher in the future

Reply:
I subscribe to the notion that no component should contain internal margins. So to support contextual positioning just extend `Box` then use `my/mx` etc as needed where it's placed. So long as the types are there I don't think rest confusing. In simpler cases like this maybe it seems like overkill, but knowing any given component ultimately supports box without a wrapper is kind of nice. Internal margins in components like these have caused a lot of annoyances (I'm thinking of `FilterArtworks` right now as a good example of this problem — constantly battling double borders and messed up margins with that thing)

Reply:
Makes total sense 👍 If you see an opportunity to standardize this somehow with a wrapper component (similar to `<Clickable>`) we should do that, but not sure what that would look like. Either way all good